meilisearch-lib/src/index/updates.rs

```diff
@@ -336,7 +336,6 @@ impl Index {
                         indexed_documents: count,
                         number_of_documents: count,
                     };
-                    info!("document addition done: {:?}", addition);
                     Ok(addition)
```

Comment:
Would it be possible to also remove the intermediate `addition` variable here, please?
```rust
Ok(DocumentAdditionResult {
    indexed_documents: count,
    number_of_documents: count,
})
```

Reply:
Thanks for  your suggestion! I am new to Rust and this is the first time I realize removing the intermediate variable will help